curriculum/challenges/english/20-upcoming-python/learn-regular-expressions-by-building-a-password-generator/656ef89b4b486b7a16077864.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 656ef89b4b486b7a16077864
+title: Step 26
+challengeType: 20
+dashedName: step-26
+---
+
+# --description--
+
+A tuple is another built-in data structure in Python. Tuples are very much like lists, but they are identified by parentheses `()`, instead of square brackets. Also, tuples are immutable, unlike lists.
```

Comment:
```suggestion
A tuple is another built-in data structure in Python. Tuples are very much like lists, but they are defined with parentheses `()`, instead of square brackets. Also, tuples are immutable, unlike lists.
```